kolibri/core/assets/src/views/breadcrumbs/index.vue

```diff
@@ -117,56 +137,53 @@
             const updatedCrumb = crumb;
             updatedCrumb.ref = crumbRefs[index];
             updatedCrumb.sensor = new ResizeSensor(updatedCrumb.ref, () => {
-              this.throttleUpdateCrumbs();
+              this.updateCrumbs();
             });
             return updatedCrumb;
           });
           this.updateCrumbs();
         });
       },
 
-      resetCollapsedState() {
-        this.crumbs = this.crumbs.map(crumb => {
-          const updatedCrumb = crumb;
-          updatedCrumb.collapsed = false;
-          return updatedCrumb;
+      detachSensors() {
+        this.crumbs.forEach(crumb => {
+          crumb.sensor.detach();
```

Comment:
this may not work as intended. I think you need to pass in a reference to the event you're detaching: https://github.com/marcj/css-element-queries/blob/master/src/ResizeSensor.js#L190

Looking at this more carefully, I think I'm doing it wrong in `responsive-element.js` by passing in both the element and the event.